src/test/java/core/basesyntax/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.impl;
+
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.ReportCreatorService;
+import core.basesyntax.service.impl.ReportCreatorServiceImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class ReportCreatorServiceImplTest {
+    private ReportCreatorService reportCreatorService = new ReportCreatorServiceImpl();
+
+    @Test
+    void createReport_nullMap_NotOK() {
+        assertThrows(RuntimeException.class, () -> reportCreatorService.createReport(null));
+    }
+
+    @Test
+    void createReport_emptyMap_NotOK() {
+        Map<String, Integer> map = new HashMap<>();
+        String report = reportCreatorService.createReport(map);
+        assertSame(15, report.length());
```

Comment:
move 15 into "expected" variable, it will improve the readability of your code. See other methods